db/madmin_conversion.py

```diff
@@ -0,0 +1,306 @@
+        `init` boolean NOT NULL,
+        `level` boolean NOT NULL,
+        `route_calc_algorithm` enum('optimized','quick') COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+        `speed` float DEFAULT NULL,
+        `max_distance` float DEFAULT NULL,
+        `ignore_spinned_stops` boolean DEFAULT NULL,
+        `cleanup_every_spin` boolean DEFAULT NULL,
+        PRIMARY KEY (`area_id`),
+        CONSTRAINT `fk_area_pokestops`
+            FOREIGN KEY (`area_id`) 
+            REFERENCES `settings_area` (`area_id`)
+            ON DELETE CASCADE
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;""",
+
+    """CREATE TABLE IF NOT EXISTS `settings_area_raids_mitm` (
+        `area_id` int(11) NOT NULL,
+        `init` boolean NOT NULL,
+        `geofence_included` varchar(256) COLLATE utf8mb4_unicode_ci NOT NULL,
+        `geofence_excluded` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+        `routecalc` varchar(256) COLLATE utf8mb4_unicode_ci NOT NULL,
+        `including_stops` boolean NOT NULL,
+        `speed` float DEFAULT NULL,
+        `max_distance` float DEFAULT NULL,
+        `delay_after_prio_event` int(11) DEFAULT NULL,     
+        `priority_queue_clustering_timedelta` float DEFAULT NULL,
+        `remove_from_queue_backlog` float DEFAULT NULL,
+        `starve_route` boolean DEFAULT NULL,
+        `init_mode_rounds` int(11) DEFAULT NULL,
+        `monlist_id` int(11) DEFAULT NULL,
+        PRIMARY KEY (`area_id`),
+        CONSTRAINT `fk_area_raids_mitm`
+            FOREIGN KEY (`area_id`) 
+            REFERENCES `settings_area` (`area_id`)
+            ON DELETE CASCADE,
+        CONSTRAINT `fk_ar_monid`
+            FOREIGN KEY (`monlist_id`) 
+            REFERENCES `settings_monivlist` (`monlist_id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;""",
+
+    """CREATE TABLE IF NOT EXISTS `settings_walker` (
+        `walker_id` int(11) NOT NULL AUTO_INCREMENT,
+        `guid` varchar(32) NULL,
+        `instance_id` int(11) NOT NULL,
+        `name` varchar(128) COLLATE utf8mb4_unicode_ci NOT NULL,
+        PRIMARY KEY (`walker_id`),
+        -- UNIQUE KEY `origin` (`name`, `instance_id`),
+        CONSTRAINT `fk_w_instance`
+            FOREIGN KEY (`instance_id`) 
+            REFERENCES `madmin_instance` (`instance_id`)
+            ON DELETE CASCADE
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;""",
+
+    """CREATE TABLE IF NOT EXISTS `settings_walkerarea` (
+        `walkerarea_id` int(11) NOT NULL AUTO_INCREMENT,
+        `guid` varchar(32) NULL,
+        `instance_id` int(11) NOT NULL,
+        `area_id` int(11) NOT NULL,
+        `algo_type` enum('countdown','timer','round','period','coords', 'idle') COLLATE utf8mb4_unicode_ci NOT NULL,
+        `algo_value` varchar(256) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+        `max_walkers` int(11) DEFAULT NULL,
+        `name` varchar(128) COLLATE utf8mb4_unicode_ci DEFAULT NULL,

```

Comment:
For consistency, `name` should be on the 4th position.

Reply:
Agreed.  Resolved in 46ad21a